admin/database/database.go

```diff
@@ -446,3 +457,8 @@ type Invite struct {
 	Role      string
 	InvitedBy string `db:"invited_by"`
 }
+
+type OrganizationDeploymentsAndSlots struct {
+	Deployments int
+	Slots       int
+}
```

Comment:
Subjective, but might be simpler to just return (int, int, error) from handler instead of having a separate type

Reply:
ok this looked cleaner as return types are explicit.